main.py

```diff
@@ -169,7 +169,18 @@ def send_email(request: Request) -> Tuple[str, int]:
         response.raise_for_status()
         log_info("notify email requested", **log_context)
     except RequestException as error:
-        notify_error = error.response.json()["errors"][0]
+        try:
+            notify_error = error.response.json()["errors"][0]
+        except AttributeError:
+            message = "no response"
+            status_code = 444
+            log_error(
+                message,
+                **log_context,
+                status_code=status_code,
+            )
+            return message, status_code
+
         status_code = error.response.status_code
```

Comment:
This needs to be moved to the `try` block also as the error was `NoneType object has no attribute json` which means it won't have the attribute `status_code` either. 

Could also use inline ifs to reduce nesting but not a blocker.